package.json

```
@@ -31,12 +31,12 @@
         "docs:review": "alex packages/**/*.md",
         "docs:start": "yarn workspace documentation serve --watch",
         "find": "test -f custom-elements.json",
-        "gen-react-wrapper": "rm -fr react && lerna exec --ignore \"{@spectrum-web-components/{base,bundle,custom-vars-viewer,modal,iconset,shared,opacity-checkerboard,styles,reactive-controllers,eslint-plugin},stylelint-header,@swc-react/*,documentation,example-project-rollup,example-project-webpack,swc-templates,@types/swc}\" -- cem analyze --config ../../cem-react-wrapper.config.js && node ./scripts/generate-icon-react-wrapper.js",
+        "gen-react-wrapper": "rm -fr react && yarn changeset version && yarn changeset publish && yarn cem-react-wrapper && node ./scripts/generate-icon-react-wrapper.js",
```

```suggestion
        "gen-react-wrapper": "rm -fr react && yarn changeset version && yarn changeset publish && yarn cem-react-wrapper && node ./scripts/generate-icon-react-wrapper.js",
```

Previously this command did not include any versioning or publish steps; is it an expected change to include those in the gen-react-wrapper step now when they weren't before?

---

True,
Also this new command for `gen-react-wrapper` is not taking into account the` --ignore ` flag as well. I guess I will have to re think on thi one.